README.md

```diff
@@ -3,22 +3,97 @@ A distributed key-value in-memory store system with table concept.
 
 Full [document](https://docs.dst-pro.com) is here.
 
-## Getting Involved
-Thank you for your attention to the `Dst` project. If you have any questions, you can create a new issue in our [Issues](https://github.com/dst-project/dst/issues) list.
-And we welcome you to participate in our `Dst` project, if you want to make some contributions, you can refer the file [CONTRIBUTING.md](https://github.com/dst-project/dst/blob/master/CONTRIBUTING.md).
+## Project Description
+`Dst` project is a memory-based distributed key-value storage system. Besides these features, dst can support table concept which looks like tables in relational databases. We use java to finish this project, which is somewhat different from most databases using c/c++.
 
-## Getting Started
-#### 1. Required environment
-JDK >= 1.8 
+## Fancy Features
```

Comment:
## Fancy Features
1. Redis-like data structure
2. Table concept based on k-v store
3. High available since this is distributed
4. Easy to use client

Reply:
Done.

Reply:
Done.